node/Cargo.lock

```diff
@@ -1956,9 +1956,9 @@ dependencies = [
 
 [[package]]
 name = "smallvec"
-version = "1.11.2"
+version = "1.11.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "4dccd0940a2dcdf68d092b8cbab7dc0ad8fa938bf95787e1b916b0e3d0e8e970"
+checksum = "942b4a808e05215192e39f4ab80813e599068285906cc91aa64f923db842bd5a"
```

Comment:
Nit: Some weird crate downgrades in the lockfile; don't seem to be related to the PR. Are they intentional?

Reply:
they are not intentional, I don't know why my resolved makes this downgrade. I've bumped the tokio version requirement to prevent the downgrade.